src/gui_gtk_x11.c

```diff
@@ -3099,6 +3099,11 @@ drawarea_configure_event_cb(GtkWidget	      *widget,
     if (event->width == cur_width && event->height == cur_height)
 	return TRUE;
 
+#if GTK_CHECK_VERSION(3,0,0)
+    if (gtk_check_version(3, 22, 2) && event->send_event == FALSE)
```

Comment:
The function version of `gtk_check_version` returns NULL when compatible,
unlike the macro counterpart that returns TRUE.
Inverting the check (prepend a `!`) made this patch functional again for me.